graylog2-web-interface/src/components/inputs/InputsList.jsx

```diff
@@ -83,26 +144,33 @@ const InputsList = createReactClass({
         </IfPermitted>
         }
 
-        <Row id="global-inputs" className="content input-list">
+        <Row id="filter-input" className="content input-list">
           <Col md={12}>
-            <h2>
+            <SearchForm onSearch={this._onFilterInputs}
+                        topMargin={0}
+                        onReset={this._onFilterReset}
+                        searchButtonLabel="Filter"
+                        placeholder="Filter by title" />
+            <br />
+            <h1>
```

Comment:
Looking at this again, `h1` feels too big. Could we make it `h2` again and increase the spacing between global and local inputs? I think that could make the section a bit less prominent, while keeping some separation between them.